services/masterdata/src/lib/core/services/masterdata-crud-http.service.ts

```diff
@@ -1,12 +1,12 @@
 import { Injectable, Inject, Optional, InjectionToken } from "@angular/core";
```

Comment:
Do you use nswag to generate the client context? This will help you identifying changed endpoints at transpile time and save you a lot of typing urls manually.

Reply:
no, i didn't use nswag for masterdata. i wrote the request in the masterdata-crud-http.service.ts

Reply:
+1

Reply:
We should also make a decision here: @Johanh-od @jstorteboom, will we use a client generator like nswag or not?

Reply:
the QueryModel will remain like this in the masterdata service for now and without using the nswag for now. we could implement this in a future update!